src/main/java/com/elastic/support/diagnostics/chain/DiagnosticChainExec.java

```diff
@@ -2,22 +2,42 @@
 
 import com.elastic.support.config.Constants;
 import com.elastic.support.diagnostics.DiagnosticException;
+import com.elastic.support.diagnostics.commands.*;
 import com.elastic.support.util.SystemProperties;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
-import java.util.List;
-import java.util.Map;
 
 public class DiagnosticChainExec {
 
     private static Logger logger = LogManager.getLogger(DiagnosticChainExec.class);
 
-    public void runDiagnostic(DiagnosticContext context, List<String> chain) {
+    public static void runDiagnostic(DiagnosticContext context, String type) {
 
         try {
-            Chain diagnostic = new Chain(chain);
-            diagnostic.execute(context);
+            if("standard".equals(type) || "remote".equals(type)) {
+                new GenerateManifestCmd().execute(context);
+                new DiagVersionCheckCmd().execute(context);
+                new VersionCheckCmd().execute(context);
+                new UserRoleCheckCmd().execute(context);
+                new RunClusterQueriesCmd().execute(context);
+            }
+
+            if("standard".equals(type)){
```

Comment:
If it's not worth a `switch` statement, then this should probably be nested under the above `if`, since it's implicitly `false` if that's `false` and 50/50 `true` if it's `true`.

Then, you can make the `"logstash"` check an `else if` and add an `else`.